### ISSUE TITLE
Playback Failed

### ISSUE THREAD

**Issue by DCWizMan** [srbaber/xbmc-easynews#55]
Getting this message on all versions now.  Kodi Omega on a 2019 updated NVidea shield.  Also tested on a different Kodi install and same issue.

**Comment by srbaber** [comment]
Hmm. Not seeing the problem.  Maybe easynews was down?  Do you have a
search term or kodi log?

I just updated to omega on my shield too

**Comment by DCWizMan** [comment]
I searched for “NHL Rangers”.  It’s not EN as that search plays in FEN-Light addon.

**Comment by DCWizMan** [comment]
Additional info; It coincidentally started happening after the manual Shield latest Hotfix update.  An other app I have called X-Plore (a file manager) no longer lets me access the Kodi data files after this hotfix.  Could the playback issue be related?

**Comment by srbaber** [comment]
I had a few last night and checked the kodi log.  Lots of timeouts from
easy news.  Even on chrome.  Waited 5 min and it finally played in Mac
chrome and then on shield.

**Comment by srbaber** [comment]
Was going to see if there is a parm I can pass to wait a little longer too

**Comment by DCWizMan** [comment]
I installed the addon on another box an ONN 4K pro.  Unfortunately, I get the same playback error message.  Searches are fine just no playback.

**Comment by DCWizMan** [comment]
Any update or is this add-on now dead for good?  I hope not.

**Comment by srbaber** [comment]
I had noticed a few more playback errors but they seem to have cleared up
over the last few days.
Unfortunately, the actual playback is done by kodi and the app simply
passes the url into the player.  There wasn't any kind of timeout / retry
parameter I could add.  i'm only getting timeouts now on the big files (>
5gb) but after a few tries they do start.   It's kind of like they have to
copy them from the archives into the active servers and bigger files just
take too long.

The only thing I've found (and maybe why mine is working a little better)
is the addition of the advancedsettings.xml.  The timeout for playback is
set to only 10 seconds.  So if easynews doesn't start responding (such as
playing back in chrome or starting the download) then kodi times out.

The file allows you to control things in kodi that aren't part of the
normal menus.

I've attached the one I've been testing out...

/sdcard/Android/data/org.xbmc.kodi/files/.kodi/userdata/advancedsettings.xml

Basically, all it does is extend the timeout and adds retries.

<advancedsetting version="1.0">
    <network>
        <curlclienttimeout>20</curlclienttimeout>
        <curllowspeedtime>40</curllowspeedtime>
        <curlretries>5</curlretries>
    </network>
</advancedsetting>

If you want to recreate the error and send me your kodi.log that would help
to confirm if it's simply an easynews timeout error or something else.

/sdcard/Android/data/org.xbmc.kodi/files/.kodi/temp/kodi.log

Have you tried turning off the VPN (assume you're using the Privado vpn)?
 There are days where it's unusable for me.

I've been experimenting with a different command to launch the player too.
But, as it's the same method of just passing in the url I don't think it's
going to change much.

**Comment by srbaber** [comment]
if you're interested, I'm getting ready to push an update soon.   want to do a little more testing.  

https://github.com/srbaber/xbmc-easynews/blob/passing-session-id/plugin.video.easynews/plugin.video.easynews-21.1.2.zip

you can pull the plugin from the branch if you want to check it out and see if it makes any difference...

i haven't been having playback issues for the past couple weeks.  anything that won't play in kodi doesn't play in chrome either.
